test/test.js

```diff
@@ -137,6 +137,17 @@ describe('ScssCommentParser', function () {
       });
     });
 
+    describe('css rule', function(){
+      it('should detect it', function(){
+        var context = parser.contextParser(getContent('rule.test.scss'));
+        assert.deepEqual(context, {
+          type: 'css',
+          name: '.foo',
+          value: 'font-weight: bold;\n  color: red;\n  .bar {\n    color: blue;\n  }'
```

Comment:
Ahh, indeed, just the fact that the code was ending with a `}` and not beginning with one was bugging me.

And the `{}` are removed for other annotations so you're right that's the right way to do. :+1: 

The mixin uses `name` and `code` so it's still consistent to use `code` for CSS rules too, though `selector` and `body` also really make sense... let's see @SassDoc/owners's opinion.